examples/tfengine/generated/org_foundation/live/audit/resources/main.tf

```diff
@@ -38,21 +40,16 @@ resource "google_organization_iam_audit_config" "config" {
   }
 }
 
-# BigQuery log sink.
-module "bigquery_log_export" {
```

Comment:
This PR is getting a bit too big. Maybe also consider do a PR that prepares adding support for folder which includes changing module to resource, etc.